### DISCUSSION TITLE
Livewire Checkbox checked="checked" with wire:model not working

### DISCUSSION THREAD

**Discussion by idproperti** [livewire/livewire#5712] [7. Bugs (archived)]
Blade

```
<div class="form-group row">
     <label for="trial" class="col-sm-2 col-form-label">Trial Checkbox</label>
           <div class="col-sm-10 container">
                <input type="checkbox" value="aaa" wire:model="trial" checked="checked"> Example1
                <input type="checkbox" value="bbb" wire:model="trial"> Example2
                <input type="checkbox" value="ccc" wire:model="trial" checked="checked"> Example3
                <input type="checkbox" value="ddd" wire:model="trial"> Example4
                <input type="checkbox" value="eee" wire:model="trial" checked="checked"> Example5
          </div>
</div>
```

Controller
`public $trial = [];`

If we change to wire:click, it works properly, which means the checkbox(es) are checked, but if we use wire:model, all of the checkbox(es) are unchecked, please try this.
The other problem is when we click the checkbox faster, sometimes it just like loading for a few seconds, and sometimes uncheck even if we already click it (if we use wire:model for the checkbox)

Thanks for your reply

**Comment by Mecit** [comment]
When working on Livewire components, instead of HTML's native `checked` functionality, you should assign the values you want to be initially checked in the `$trial` property. e.g `public $trial = ['aaa', 'ccc', 'eee']` will have the checkboxes with the given values checked when the page loads. Alternatively, you can do that in the `mount()` method of the component, too.

```
public function mount() {
    $this->trial =  ['aaa', 'ccc', 'eee'];
}
```

**Reply by idproperti** [reply]
It's still doesn't work, already change to string

**Comment by isaacangello** [comment]
in livewire 3 use wire:model.live to update your var into to the class
